mozilla/fx-private-relay
# fix MPP-3123: include per-language hero images

This PR fixes #MPP-3123.

How to test:

1. Set up end-to-end email, or use the dev server
2. Set you browser to one of the 16 languages of the images in this PR
3. Sign up for a new Relay account
4. Check the email inbox of the FXA
   * [ ] The email header "hero image" should be in the language matching the language of the browser when you signed up
